src/vm/comutilnative.h

```diff
@@ -122,7 +122,8 @@ class GCInterface {
     static FCDECL1(int,     GetGenerationWR, LPVOID handle);
     static FCDECL1(int,     GetGeneration, Object* objUNSAFE);
     static FCDECL0(UINT64,  GetSegmentSize);
-
+    static FCDECL0(int,     GetLastGCPercentTimeInGC);
+    static FCDECL1(size_t,  GetGenerationSize, int gen);
```

Comment:
The FCall signature needs to match between C++ and C#. The valid combinations are:
`size_t` in C++ and `UIntPtr` in C#
`INT64` n C++ and `long` in C#